other/concat-exports.js

```diff
@@ -0,0 +1,26 @@
+const htmlTagNames = require('html-tag-names')
+const svgTagNames = require('svg-tag-names')
+const appendToFile = require('fs').appendFile
+const resolve = require('path').resolve
+
+const capitalize = s => s.slice(0, 1).toUpperCase() + s.slice(1)
+
+const dashToCamelCase = s =>
+  s.replace(/-([a-z])/g, ([, char]) => char.toUpperCase())
+
+const elementExports = htmlTagNames
+  .concat(svgTagNames)
+  .filter((tag, index, array) => array.indexOf(tag) === index)
+  .map(tag => {
+    const validJsName = dashToCamelCase(tag)
+    const capitalName = capitalize(validJsName)
+    return `export const ${capitalName} = glamorous['${tag}']`
```

Comment:
Let's add a semicolon here because this is our transpiled code, we should never rely on ASI.